src/FasterImage/ImageParser.php

```diff
@@ -133,7 +135,13 @@ public function parseType()
                 case "MM":
                     return $this->type = 'tiff';
                 default:
-                    return false;
+                    $this->stream->resetPointer();
+                    $markup = $this->stream->read( 1024 );
```

Comment:
That's a good question. Mainly I was thinking to read a full kilobyte because it would be surely guaranteed that the `<svg>` element would be inside of the first 1KB.

And this avoids having to keep looking for `<svg` multiple many times as the size of the buffer keeps growing. This is more important below when we need to obtain the entire `<svg>` element.

I have another idea to try that won't require changing `Stream`.